src/@types/assets/index.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module '*.svg' {
```

Comment:
Afaik, you don't need to declare this as a `*.d.ts` which requires us to alter typeRoots.

Can we just include this somewhere?

```
declare module '*.svg' {
  const content: any;
  export default content;
}
```

Also what are the react props for?  `svg-url-loader` imports raw svg right/